src/quo/components/list_items/network_list/component_spec.cljs

```diff
@@ -0,0 +1,43 @@
+(ns quo.components.list-items.network-list.component-spec
+  (:require [quo.components.list-items.network-list.view :as network-list]
+            [quo.foundations.colors :as colors]
+            [test-helpers.component :as h]))
+
+(def props
+  {:state               :transparent
+   :label               "Mainnet"
+   :network-name        :ethereum
+   :customization-color :blue
+   :token-value         "100.00 ETH"
+   :fiat-value          "€100.00"})
```

Comment:
The tests are failing since `:theme` now is mandatory and is missing from here:
``` clj
     Invalid function arguments:
       [{:customization-color :blue,
         :fiat-value "€100.00",
         :label "Mainnet",
         :network-name :ethereum,
         :state :active,
         :theme nil,
         ...}]
 
     Input Schema:
       [:catn [:props [:map {#} [#] [#] [#] [#] ...]]]

     Errors:
       {:in [0 :theme],
        :message "should be either :light or :dark",
        :path [:props :theme 0],
        :schema [:enum :light :dark],
        :value nil}
```

Reply:
thanks @clauxx just going through the feedback now, will fix this during the updates